platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/builder/PlatformBuilder.java

```diff
@@ -143,14 +146,15 @@ public final class PlatformBuilder {
      * the app will pass the loaded state via the initialState argument to this method. If the snapshot doesn't exist,
      * then the app will create a new genesis state and pass it via the same initialState argument.
      *
-     * @param appName             the name of the application, currently used for deciding where to store states on
-     *                            disk
-     * @param swirldName          the name of the swirld, currently used for deciding where to store states on disk
-     * @param selfId              the ID of this node
-     * @param softwareVersion     the software version of the application
-     * @param initialState        the initial state supplied by the application
+     * @param appName                  the name of the application, currently used for deciding where to store states on
+     *                                 disk
+     * @param swirldName               the name of the swirld, currently used for deciding where to store states on
+     *                                 disk
+     * @param selfId                   the ID of this node
+     * @param softwareVersion          the software version of the application
+     * @param initialState             the initial state supplied by the application
```

Comment:
Could we not do this alignment of param text (with associated frequent reformats and excessive wrapping)?  It's fine to just have a single space after the parameter name in all cases; this all gets rendered (even IDEs render the HTML view), so there's no reason to make the code "pretty".